include/SDL3/SDL_gpu.h

```diff
@@ -394,10 +396,11 @@ typedef enum SDL_GPUBlendFactor
 } SDL_GPUBlendFactor;
 
 typedef Uint8 SDL_GPUColorComponentFlags;
-#define SDL_GPU_COLORCOMPONENT_R    (1u << 0)
-#define SDL_GPU_COLORCOMPONENT_G    (1u << 1)
-#define SDL_GPU_COLORCOMPONENT_B    (1u << 2)
-#define SDL_GPU_COLORCOMPONENT_A    (1u << 3)
+
+#define SDL_GPU_COLORCOMPONENT_R (1u << 0) /**< & for the Red component */
```

Comment:
Why is there an ampersand on all of these?

Reply:
I was using the convention established by the Pen API. https://github.com/libsdl-org/SDL/blob/df501040fd646a1f4eab92aa5b698b35a36013ed/include/SDL3/SDL_pen.h#L69 I interpreted the ampersands as hints that you could AND the values together into a bit mask. (Incidentally there’s not a newline between the typedef and the defines there, which is why I didn’t include them initially!)

Reply:
Ah, yeah, that needs to be fixed. SDL_render.h and SDL_video.h are better examples of the SDL convention.

Reply:
SDL_pen.h is now fixed. :)